Jenkinsfile

```diff
@@ -54,8 +54,8 @@ pipeline {
   }
   parameters {
     booleanParam(name: 'runAllStages', defaultValue: false, description: 'Allow to run all stages.')
-    booleanParam(name: 'windowsTest', defaultValue: true, description: 'Allow Windows stages.')
-    booleanParam(name: 'macosTest', defaultValue: true, description: 'Allow macOS stages.')
+    booleanParam(name: 'windowsTest', defaultValue: false, description: 'Allow Windows stages.')
```

Comment:
we have immutable windows workers, Why do not test windows?

Reply:
That's a good point, somehow I thought it was not required  then I reverted, but I did not commit it :/